packages/browser/src/login/oidc/oidcHandlers/AuthorizationCodeWithPkceOidcHandler.ts

```diff
@@ -86,48 +86,45 @@ export default class AuthorizationCodeWithPkceOidcHandler
+        // We use the OAuth 'state' value (which should be crypto-random) as
+        // the key in our storage to store our actual SessionID. We do this
+        // 'cos we'll need to lookup our session information again when the
+        // browser is redirected back to us (i.e. the OAuth client
+        // application) from the Authorization Server.
+        // We don't want to use our session ID as the OAuth 'state' value, as
+        // that session ID can be any developer-specified value, and therefore
+        // may not be appropriate (since the OAuth 'state' value should really
+        // be an unguessable crypto-random value).
+        // eslint-disable-next-line no-underscore-dangle
+        storage.setForUser(signingRequest.state._id, {
+          sessionId: oidcLoginOptions.sessionId,
+        }),
+
+        // Store our login-process state using the session ID as the key.
+        // Strictly speaking, this indirection from our OAuth state value to
+        // our session ID is unnecessary, but it provides a slightly cleaner
+        // separation of concerns.
+        storage.setForUser(oidcLoginOptions.sessionId, {
           // eslint-disable-next-line no-underscore-dangle
-          storage.setForUser(req.state._id, {
-            sessionId: oidcLoginOptions.sessionId,
-          }),
+          codeVerifier: signingRequest.state._code_verifier,
+          issuer: oidcLoginOptions.issuer.toString(),
+          // The redirect URL is read after redirect, so it must be stored now.
+          redirectUrl: oidcLoginOptions.redirectUrl,
+          dpop: oidcLoginOptions.dpop ? "true" : "false",
+        }),
+      ]);
 
-          // Store our login-process state using the session ID as the key.
-          // Strictly speaking, this indirection from our OAuth state value to
-          // our session ID is unnecessary, but it provides a slightly cleaner
-          // separation of concerns.
-          storage.setForUser(oidcLoginOptions.sessionId, {
-            // eslint-disable-next-line no-underscore-dangle
-            codeVerifier: req.state._code_verifier,
-            issuer: oidcLoginOptions.issuer.toString(),
-            // The redirect URL is read after redirect, so it must be stored now.
-            redirectUrl: oidcLoginOptions.redirectUrl,
-            dpop: oidcLoginOptions.dpop ? "true" : "false",
-          }),
-        ])
-          .then(() => {
-            redirector.redirect(req.url.toString(), {
-              handleRedirect: oidcLoginOptions.handleRedirect,
-            });
-          })
-          // eslint-disable-next-line @typescript-eslint/no-explicit-any
-          .catch((err: unknown) => {
-            // eslint-disable-next-line no-console
-            console.error(err);
-          })
-      );
-    });
+      redirector.redirect(signingRequest.url.toString(), {
+        handleRedirect: oidcLoginOptions.handleRedirect,
+      });
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I think with `unknown` this might not be necessasry? (But be sure to check - might just be an ESLint warning, causing GitHub to add a million "check annotations" again...)

```suggestion
```

Reply:
Fixed in https://github.com/inrupt/solid-client-authn-js/pull/1243/commits/18980ee5072d3c6a207bd715908ae77c2f2860fc